conn.go

```diff
@@ -1,5 +1,10 @@
+//+build !windows
```

Comment:
I think @christian-roggia advice here has been taken a little too literally. Use whatever code works on both unix and windows targets, I suspect your sizeof(WhateverAny) will work on both.